app/views/votes/create.html.erb

```diff
@@ -0,0 +1 @@
+<h1>Your vote has been recorded!</h1>
```

Comment:
@joelbyler cool. Are the students redirecting to the /votes/thanks-for-voting controller (perhaps either after a delay, or just sending them to this page on successful submit so that it flows to West Shore's (thanks-for-voting) page?